.github/workflows/promote.yml

```diff
@@ -0,0 +1,116 @@
+
+name: Promote to channel
+
+env:
+  ORG: kyma-project
+  BTP_MANAGER_REPO: btp-manager
+  KYMA_REPO: kyma
+  TEMPLATE_NAME: moduletemplate-btp.yaml
+  GIT_EMAIL: <>
+  GIT_NAME: BTP Manager
+
+on:
+  workflow_dispatch:
+    inputs:
+      releaseTag:
+        description: 'Release Tag'
+        default: use latest available release 
+        required: true
+      channel:
+        description: 'Channel'
+        type: choice
+        options:
+        - alpha
+        - fast
+        - regular
+        required: true
+
+jobs:
+  promote:
+    runs-on: ubuntu-latest
+
+    steps:
+
+    - name: Use default Release Tag
```

Comment:
Possible we can literally mention in the step name, that `default` is `latest`.
E.g. : -name: Use default (latest) Release Tag